src/tests/ftest/io/macsio_test.py

```diff
@@ -80,7 +80,7 @@ def test_macsio_daos_vol(self):
         Use case:
             Six clients and two servers.
 
-        :avocado: tags=all,pr,hw,large,io,macsio_daos_vol,DAOS_5610
+        :avocado: tags=all,daily_regression,hw,large,io,macsio_daos_vol,DAOS_5610
```

Comment:
This needs to be broken up into multiple lines.

In general though, as multi-line tag definitions are becoming more prevalent maybe it makes sense to require multi-line tag definitions for all tests to provide a cleaner break down.  Something like:
```
        :avocado: tags=<ci_build_tags>
        :avocado: tags=<ci_stage_tags>
        :avocado: tags=<test_identification_tags>   [e.g. <dir>,<class>,<method>]
        :avocado: tags=<ticket_identification_tags> [OPTIONAL]
```
In this case that would look like this:
```suggestion
        :avocado: tags=all,daily_regression
        :avocado: tags=hw,large
        :avocado: tags=io,macsio,macsio_daos_vol
        :avocado: tags=DAOS_5610
```

Reply:
Done.